data-extraction/recipes.json

```diff
@@ -0,0 +1,778 @@
+[
+    {
+        "id": 1,
+        "spoonacular-id": 333606,
+        "title": "Halibut with Lemon, Capers, and Croutons",
+        "image": "https://spoonacular.com/recipeImages/halibut-with-lemon-capers-and-croutons-333606.jpeg",
+        "ingredients": [
+            {
+                "amount": 1.0,
+                "id": 1001,
```

Comment:
this id is the spoonacular id (hadn't flubbed an id for our uses yet since we don't have most of these ingredients in our test data). We can flub our own unique ids for these as well and these should at the very least be renamed to spoonacular-id to be consistent. Will fix and will update when fixed